### PR TITLE
Maxime/checkpoints

### PR DESCRIPTION
**Pull request by Franchovy**

### CODE REVIEW

File: src/sprites/AbilityPanel.lua [hunk 1]
```diff
@@ -47,103 +47,56 @@ function AbilityPanel.getInstance() return _instance end
...
   end
 end
 
-function AbilityPanel:cleanUp()
-  self:setItems()
-
-  self:remove()
-end
+function AbilityPanel:remove()
+  AbilityPanel.super.remove(self)
```
Comment (CMorooney) [inline src/sprites/AbilityPanel.lua:86]: should this be called after the loop that removes all the children? or does it not really matter

File: src/sprites/AbilityPanel.lua [hunk 2]
```diff
@@ -47,103 +47,56 @@ function AbilityPanel.getInstance() return _instance end
...
+function AbilityPanel:updateBlueprints()
+  local blueprints = Player.getInstance().blueprints
+  self.blueprints = blueprints
 
-function AbilityPanel:updateItemsCount()
-  self.abilitiesCount = #items
-end
+  for i, sprite in ipairs(buttonSprites) do
+    if blueprints[i] then
+      sprite:add()
```
Comment (CMorooney) [inline src/sprites/AbilityPanel.lua:139]: aren't the sprites already added when AbilityPanel is [here](https://github.com/Franchovy/PlayJam-5/pull/2/files#diff-7f6dc5a823571736a8d4c4e52c74b697c0e6b75cde2bc4edfb5a9d78ce223842R74)?

**Review by CMorooney — APPROVED** [review]

File: src/sprites/AbilityPanel.lua (hunk 1 above)
Comment (Franchovy) [reply src/sprites/AbilityPanel.lua:86]: Doesn't really matter... but we can do it for cleanliness :)

File: src/sprites/AbilityPanel.lua (hunk 2 above)
Comment (Franchovy) [reply src/sprites/AbilityPanel.lua:139]: Point valid, I'll make sure to correct that

**Pull request merged**